addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientListScreen.js

```diff
@@ -48,7 +48,6 @@ odoo.define('point_of_sale.ClientListScreen', function(require) {
                     }
                 },
             };
-            this.updateClientList = debounce(this.updateClientList, 70);
```

Comment:
Good catch. But, can we keep the debouncing of the method?

Reply:
Why? Search in js cache + render works fast enough, not?

Reply:
I think it's not difficult to reach a number of cached contacts that will throttle the typing in the search bar. Besides, 70ms is not a terrible response time when typing in a search bar.

Reply:
70ms is still sensible difference. But you are right and the search may work slowly with big number of cached contacts and slow devices. 

I've reverted removing debouning